.ado/jobs/integration-test.yml

```diff
@@ -5,125 +5,160 @@ parameters:
-      - powershell: Start-Sleep -s 5
-        displayName: Pause before screenshot
-
-      - script: yarn take-screenshot --filename $(Build.StagingDirectory)/Screenshots/before-test.png
-        displayName: Screenshot desktop before tests
-        condition: succeededOrFailed()
-
-      - script: yarn integration-test
-        displayName: yarn integration-test
-        workingDirectory: packages/integration-test-app
-        condition: and(succeeded(), eq(variables.NoDeployOption, ''))
-
-      - script: yarn take-screenshot --filename $(Build.StagingDirectory)/Screenshots/after-test.png
-        displayName: Screenshot desktop after tests
-        condition: succeededOrFailed()
-
-      - task: PublishPipelineArtifact@1
-        displayName: Upload targets
-        inputs:
-          artifactName: Targets - IntegrationTest-$(BuildPlatform)-$(System.JobAttempt)
-          targetPath: packages/integration-test-app/windows/$(BuildPlatform)/Debug/integrationtest
-        condition: failed()
-
-      - task: PublishPipelineArtifact@1
-        displayName: Upload screenshots
-        inputs:
-          artifactName: Screenshots - IntegrationTest-$(BuildPlatform)-$(System.JobAttempt)
-          targetPath: $(Build.StagingDirectory)/Screenshots
-        condition: succeededOrFailed()
-
-      - template: ../templates/upload-build-logs.yml
-        parameters:
-          buildLogDirectory: '$(BuildLogDirectory)'
-          condition: succeededOrFailed()
+            BuildConfiguration: Debug
+            DeployOptions:
+            UseHermes: false
+
+jobs:
+  - ${{ each config in parameters.buildMatrix }}:
+    - ${{ if eq(config.BuildEnvironment, parameters.buildEnvironment) }}:
+      - ${{ each matrix in config.Matrix }}:
+        - job: IntegrationTest${{ matrix.Name }}
+          displayName: Integration Test App ${{ matrix.Name }}
+          variables:
+            - template: ../variables/vs2019.yml
+
+          pool:
+            vmImage: $(VmImage)
+
+          timeoutInMinutes: 60 # how long to run the job before automatically cancelling
+          cancelTimeoutInMinutes: 5 # how much time to give 'run always even if cancelled tasks' before killing them
+
+          steps:
+            - checkout: self
+              clean: true
+              submodules: false
+                  
+            - template: ../templates/prepare-env.yml
+
+            - powershell: |
+                Write-Host "##vso[task.setvariable variable=BuildLogDirectory]$(Build.BinariesDirectory)\${{ matrix.BuildPlatform }}\BuildLogs"
+              displayName: Set BuildLogDirectory
+
+            - script: .ado\scripts\SetupLocalDumps.cmd integrationtest
+              displayName: Set LocalDumps
+
+            - ${{ if eq(matrix.UseHermes, 'true') }}:
+              - powershell: (Get-Content .\windows\ExperimentalFeatures.props) -replace 'false','true' | Set-Content .\windows\ExperimentalFeatures.props
```

Comment:
>place 'false','tr [](http://example.com/codeflow?start=82&length=17)

Should this filter the line to just Hermes?
If we add more features later, they will be flipped true in this test as well...

Reply:
Yeah, it is probably better to be future proof here. Wanted to be a bit lazy to avoid xml parsing, but I should be able to just change the find/replace.